tools/serverpod_cli/lib/src/test_util/builders/generator_config_builder.dart

```diff
@@ -16,6 +17,7 @@ class GeneratorConfigBuilder {
   List<ModuleConfig> _modules;
   List<TypeDefinition> _extraClasses;
   List<ServerpodFeature> _enabledFeatures;
+  final List<ExperimentalFeature> _enabledExperimentalFeatures;
```

Comment:
Question: Shouldn't all of the lists here be final? If this is not used for `_enabledFeatures` i would suggest removing the final.

Reply:
I investigated it :) 

the linter wanted final only with _enabledExperimentalFeatures because the other fields are reassigned like:

```dart
GeneratorConfigBuilder withPackageType(PackageType type) {
    _type = type;
    return this;
  }
```

I on the other hand initially just added the passed feature to the list 

```dart
  GeneratorConfigBuilder withEnabledExperimentalFeatures(
      List<ExperimentalFeature> features) {
    _enabledExperimentalFeatures.addAll(features);
    return this;
  }
```

I solved it by also assigning a list to `_enabledExperimentalFeatures`